package.json

```diff
@@ -25,8 +25,7 @@
   "homepage": "https://github.com/PlaceMe-SAS/single-spa-angular-cli#readme",
   "dependencies": {
     "@types/core-js": "^0.9.45",
-    "rxjs": "^5.5.6",
-    "single-spa": "^3.8.1"
```

Comment:
Removed the dependency on single-spa so as to avoid having multiple instances of single-spa loaded in the same page.

This is not necessary for people who make single-spa a [webpack external](https://webpack.js.org/configuration/externals/) in their apps, to ensure that there is only one instance of single-spa. But for those who are not, a dependency on single-spa will result in multiple instances of single-spa being loaded.

If you ever add this dependency back, it might be better to be a `peerDependency` (although that still could have the same issue). And also you would have to add documentation, instructing people to make single-spa be a webpack external

Reply:
Hoo ok, I don't know...